pulp_python/app/serializers.py

```diff
@@ -3,6 +3,7 @@
 import tempfile
 
 from django.core.files.storage import default_storage as storage
+from dynaconf import settings
```

Comment:
If this works, that's neat :)

But for consistency lets use `from django.conf import settings`